userTests/typescript-eslint/test.json

```diff
@@ -0,0 +1,4 @@
+{
+  "cloneUrl": "https://github.com/typescript-eslint/typescript-eslint.git",
```

Comment:
Theoretically, I would think that this is enough to capture everything, right?

(Also, could you to ts-eslint add something that can install the latest TS and check it so you get early warnings too?)

Reply:
> Theoretically, I would think that this is enough to capture everything, right?

Our `git clone` -> `yarn install` & `yarn build` isn't what's failing in CI though. It's the `yarn test`. Does the automation also run tests? (sorry if I missed this, it would explain what you've been trying to convey to me)

> (Also, could you to ts-eslint add something that can install the latest TS and check it so you get early warnings too?)

Absolutely: https://github.com/typescript-eslint/typescript-eslint/issues/7093

Reply:
The automation only installs without scripts and runs `tsc` on each `tsconfig.json`. It never executes code from within the repo, unless you write a script-based test which would do that. If `yarn test` takes a large amount of time, I don't think we would add it here, but I would me more interested in why things fail only in `test` and not otherwise... You're still referring to an OOM, right? Not trying to catch errors in the typescript API?

Reply:
> still referring to an OOM, right? 

Exactly. I don't know why it's OOMing yet, but at the very least I'm hoping that running a similar bit of code to the offending tests would catch similar issues.